src/site/content/en/blog/origin-trials/index.md

```diff
@@ -38,21 +38,28 @@ trials, as learnings are incorporated and adjustments are made.
 ## How to register for an origin trial
 
 1. Choose an origin trial from the [list of active trials](https://developers.chrome.com/origintrials/#/trials/active).
-1. [Request a token](https://developers.chrome.com/origintrials/#/trials/active).
-1. [Add the token](http://googlechrome.github.io/OriginTrials/developer-guide.html)
-   to your web pages, using a `<meta>` tag or an `Origin-Trial` HTTP response
-   header.
+1. Request a token by clicking the **Request** button and filling out the form.
+1. Add the token to your web pages,
+   using a `<meta>` tag or an `Origin-Trial` HTTP response header.
    -  As a meta tag in the &lt;head&gt; of each page served:   
       `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
    -  As an HTTP header:  
       `Origin-Trial: TOKEN_GOES_HERE`
-
 1. Try out the new feature.
 1. Submit feedback. Do this through the origin trial site. This feedback is 
    not public and is available only to a limited group of people on the Chrome
    team. Each trial also provides a link for spontaneous community feedback.
    This typically points to the the feature on GitHub or some other public
-   channel.  
+   channel.
+1. When your token expires, you will get an email with a link to renew it.
+   In order to do so, you are again asked to submit feedback.
+   
+{% Aside 'warning' %}
+  Please note that even if an API eventually lands unchanged after a successful origin trial,
+  there is an expected short period of time between the end date of the origin trial and the
+  date where the implementation ships in the browser where the API will not be available.
```

Comment:
Maybe '... a short period of time, between the end of the origin trial and the date the implementation ships in the browser, when the API will not be available.'

Reply:
Ça! Thanks.